### DISCUSSION TITLE
Jerk Control of Floating Base Systems with Contact-Stable Parametrised Force Feedback

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#453] [Research news]
Latest achievements of [DIC@IIT](https://dic.iit.it) group in the whole-body control of iCub.

https://user-images.githubusercontent.com/3738070/102606646-53c6ce80-4127-11eb-9d2d-8674284ee9ed.mp4

The video is available on [YouTube 🎥](https://www.youtube.com/watch?v=eIFrZcbjh-I).

cc @gabrielenava @ahmadgazar @fjandrad @DanielePucci

**Comment by DanielePucci** [comment]
Thanks @pattacini !!! You just taught me how to include youtube videos in GitHub comments!

@robotology/iit-dynamic-interaction-control
